cw_core/lib/crypto_currency.dart

```diff
@@ -67,6 +67,30 @@ class CryptoCurrency extends EnumerableItem<int> with Serializable<int> implemen
     CryptoCurrency.uni,
     CryptoCurrency.stx,
     CryptoCurrency.btcln,
+    CryptoCurrency.shib,
+    CryptoCurrency.aave,
+    CryptoCurrency.arb,
+    CryptoCurrency.bat,
+    CryptoCurrency.comp,
+    CryptoCurrency.cro,
+    CryptoCurrency.ens,
+    CryptoCurrency.ftm,
+    CryptoCurrency.frax,
+    CryptoCurrency.gusd,
+    CryptoCurrency.gtc,
+    CryptoCurrency.grt,
+    CryptoCurrency.ldo,
+    CryptoCurrency.nexo,
+    CryptoCurrency.cake,
+    CryptoCurrency.pepe,
+    CryptoCurrency.storj,
+    CryptoCurrency.tusd,
+    CryptoCurrency.wbtc,
+    CryptoCurrency.weth,
+    CryptoCurrency.zrx,
+    CryptoCurrency.dydx,
+    CryptoCurrency.steth,
+    CryptoCurrency.1inch,
```

Comment:
you can't have a variable name starting with a digit, you can rename it to `oneInch` maybe

Reply:
I'll need to transform it later for sending to the exchanges, will that be okay? I can try Monday :)

Reply:
yeah, as long as they are in a quoted string ("1inch", or '1inch') then they are good to go
just the variable name is the one to be changed nothing else

have a great weekend ^^